src/test/java/create_task_use_case/CreateTaskInteractorTest.java

```diff
@@ -0,0 +1,41 @@
+package create_task_use_case;
+
+import data_access.DataAccessDay;
+import data_access.DataAccessTask;
+import entities.TaskFactory;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class CreateTaskInteractorTest {
+    private final static CreateTaskOutputBoundary createTaskPresenter = new CreateTaskPresenter();
+    private final static CreateTaskDataAccess createTaskDataAccess = new CreateTaskDataAccess();
+    private final static TaskFactory taskFactory = new TaskFactory();
+    @Test
+    public void createTestSuccess(){
+        CreateTaskInputData inputData = new CreateTaskInputData("Finish Unit Test", 3);
+        CreateTaskInteractor createTaskInteractor = new CreateTaskInteractor(taskFactory, createTaskPresenter,
+                createTaskDataAccess);
+        CreateTaskOutputData createTaskOutputData = createTaskInteractor.create(inputData);
+        DataAccessDay day = createTaskDataAccess.getDays().get(3);
+        HashMap<String, DataAccessTask> tasks = day.getTasks();
+        DataAccessTask refrenceTask = tasks.get("Finish Unit Test");
```

Comment:
typo in "refrence" (missing an e). Should be referenceTask

Reply:
oops! Good Catch!